shellsupport/bash/bash.go

```diff
@@ -19,6 +19,7 @@ func bashShellHandler(c credentials.AWSCredential, export bool) []string {
 	}
 
 	return []string{
+		fmt.Sprintf("%s AWS_PROFILE=%s;", flags, c.AWSProfile),
```

Comment:
I can try to do this but... I'm not a go developer. Just wanted to share a little hack.